### PR TITLE
Do not raise an error when there is an issue with the crossing guard record

### PR DESCRIPTION
**Pull request by chiaberry**
the knack banner ETL alerts an error every day because of crossing guard emails, this is a hopefully temporary fix to skip over crossing guard email duplications (but still log them). 

I am concerned that other non crossing guard errors will happen on the ETL and I will not catch them under the assumption its the same errors as before.

### CODE REVIEW

File: update_employees.py [hunk 1]
```diff
@@ -431,9 +430,13 @@ def main():
...
-                errors_list = e.response.json()["errors"]
-                result["errors"].append(format_errors(errors_list, record))
-                continue
+                if record["field_230"] == "Crossing Guard":
+                    logging.info(f"Error with Crossing Guard record {record[email_field]['email']}, skipped record {method}")
+                    continue
+                else:
+                    errors_list = e.response.json()["errors"]
+                    result["errors"].append(format_errors(errors_list, record))
+                    continue
```
Comment (chiaberry) [inline update_employees.py:439]: Another option would be to skip adding crossing guard records to the payload entirely

Comment (johnclary) [reply update_employees.py:439]: skipping this edge case seems best to me. i guess we'll know when this is a problem when HR asks us why some crossing guard can't do their SSPR?

**Review by johnclary — APPROVED** [review]
thank you!

**Review by Charlie-Henry — APPROVED** [review]
shush the error messages 🤫 🚢

File: update_employees.py (hunk 1 above)
Comment (chiaberry) [reply update_employees.py:439]: crossing guards SSPRs??

**Pull request merged**